test/dynamo/test_allow_inline_skip.py

```diff
@@ -0,0 +1,47 @@
+# Owner(s): ["module: dynamo"]
+import unittest
+
+import torch
+import torch._dynamo.test_case
+from torch._dynamo.utils import istype
+
+
+class AllowInlineSkipTests(torch._dynamo.test_case.TestCase):
+    def test_func_inlinelist(self):
```

Comment:
It's not clear to me what this test is trying to exercise

Reply:
I'm testing if developers adding functions to ```FUNC_INLINELIST```, it should work as expected (inlining).